onnxruntime/core/framework/utils.cc

```diff
@@ -633,12 +635,24 @@ void DumpNodeOutputs(OpKernelContext& context, const Node& node, const SessionSt
             if (!provider) {
               provider = cpu_execution_provider;
             }
+            std::cout << " Provider=" << provider->Type() << "\n";
 
```

Comment:
Can we just dump this in the 'provider == cuda_execution_provider' branch? Seems like noise unless CUDA is enabled

Reply:
Sure. I'll change it.